src/page-multi-facility/types.tsx

```diff
@@ -19,7 +20,7 @@ type Timestamp = {
 export type Facilities = Facility[];
 
 export type Scenario = {
-  id?: string;
+  id: string;
```

Comment:
I'm attempting to view these types as truly hydrated representations of what we have in Firestore.  We'll see how well that works long term